interlok-rest-health-check/src/test/java/com/adaptris/rest/healthcheck/WorkflowHealthCheckComponentTest.java

```diff
@@ -253,7 +252,7 @@ public void testReadiness_Ready() throws Exception {
     private JmxMBeanHelper mockJmxHelper;
 
     public MockedHealthCheckWrapper() throws Exception {
-      MockitoAnnotations.initMocks(this);
+      MockitoAnnotations.openMocks(this);
```

Comment:
I wonder if it's not more relevant to do:

```
    private JmxMBeanHelper mockJmxHelper;

    public MockedHealthCheckWrapper() throws Exception {
      mockJmxHelper = Mockito.mock(JmxMBeanHelper);
    }
```

instead of

```
    @Mock
    private JmxMBeanHelper mockJmxHelper;

    public MockedHealthCheckWrapper() throws Exception {
      MockitoAnnotations.initMocks(this);
    }
```

as the initMock is never closed and that the annotation actually add complexity here.

You would even not need the build method and just use the constructor?

Other than that, it's all good and I'll approve if you disagree with me :)